wrangler-service/src/main/java/io/cdap/wrangler/service/directive/AbstractDirectiveHandler.java

```diff
@@ -118,7 +121,8 @@ protected <E extends Exception> List<Row> executeDirectives(
       String namespace,
       List<String> directives,
       List<Row> sample,
-      GrammarWalker.Visitor<E> grammarVisitor) throws DirectiveParseException, E, RecipeException {
+      GrammarWalker.Visitor<E> grammarVisitor,
+      Workspace workspace) throws DirectiveParseException, E, RecipeException {
```

Comment:
Do we need to add whole workspace as an argument. Can we use just nullabilityMap instead as the argument ?

Reply:
I need the workspace object, for the subsequent changes

Reply:
Removing the workspace object as it's not needed in subsequent changes.